src/main/java/fi/metatavu/metamind/nlp/TrainingMaterialController.java

```diff
@@ -250,6 +251,29 @@ public TrainingMaterial findTrainingMaterialById(UUID trainingMaterialId) {
   public List<TrainingMaterial> listTrainingMaterials(Story story, TrainingMaterialType type, TrainingMaterialVisibility visibility) {
     return trainingMaterialDAO.list(true, story, type, visibility);
   }
+  
+  /**
+   * Lists all training materials in a given story
+   * 
+   * @param story
+   * @return training materials
+   */
+  public List<TrainingMaterial> listTrainingMaterials(Story story) {
+    List<TrainingMaterial> trainingMaterials = new ArrayList<>();
+    trainingMaterials.addAll(listTrainingMaterials(story, TrainingMaterialType.INTENTOPENNLPDOCCAT, TrainingMaterialVisibility.LOCAL));
```

Comment:
Change this to list all training materials within the story instead of listing them by the type

Reply:
Moved